src/pages/index.jsx

```diff
@@ -10,6 +10,12 @@ import MembersGridList from '../members/grid-list';
 
 export const query = graphql`
   query IndexQuery($language: String!) {
+    homeIntro: allWordpressPage(filter: {title: {eq: "Home"}, language: { eq: $language } }
```

Comment:
Should be `wordpressPage`.

Reply:
i needed all pages to filter by language in wordpressPage there is no language

Reply:
This works.

```js
query MyQuery {
  wordpressPage(language: {eq: "nl"}) {
    title
  }
}
```

Reply:
Note that the filter arg works different on single-item queries! It's not wrapped in `filter: {}`

Reply:
indeed fixed now thanks 👍